src/Simplify.elm

```diff
@@ -911,12 +911,12 @@ fromModuleToProject : Rule.ContextCreator ModuleContext ProjectContext
 fromModuleToProject =
     Rule.initContextCreator
         (\moduleContext ->
-            { initialContext
-                | exposedVariants =
-                    Dict.singleton moduleContext.moduleName
-                        (moduleContext.exposedVariants
-                            |> Dict.foldl (\_ -> Set.union) Set.empty
-                        )
+            { customTypesToReportInCases = initialContext.customTypesToReportInCases
```

Comment:
This should be `Set.empty` to reduce the memory footprint (especially now with the file-system cache).

In practice, I never reference the initial context except in the rule's definition, to avoid it changing the behavior in other places.